pkg/fission-cli/logdb/influxdb.go

```diff
@@ -66,13 +66,18 @@ func (influx InfluxDB) GetLogs(filter LogFilter) ([]LogEntry, error) {
 	parameters["time"] = timestamp
 	//the parameters above are only for the where clause and do not work with LIMIT
 
+	orderCondition := ""
+	if filter.Reverse {
+		orderCondition = " order by \"time\" desc"
+	}
+
 	if filter.Pod != "" {
 		// wait for bug fix for fluent-bit influxdb plugin
-		queryCmd = "select * from /^log*/ where (\"funcuid\" = $funcuid OR \"kubernetes_labels_functionUid\" = $funcuid) AND \"pod\" = $pod AND \"time\" > $time LIMIT " + strconv.Itoa(filter.RecordLimit)
+		queryCmd = "select * from /^log*/ where (\"funcuid\" = $funcuid OR \"kubernetes_labels_functionUid\" = $funcuid) AND \"pod\" = $pod AND \"time\" > $time " + orderCondition + "LIMIT " + strconv.Itoa(filter.RecordLimit)
```

Comment:
Add a blank before `LIMIT`

`+ "LIMIT "` -> `+ " LIMIT "`

Reply:
I found it, thanks